Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -283,9 +325,19 @@ public function getParameters()
                     'piwik.js' => 'piwik.js',
                     'js/' => 'js/',
                     'js/tracker.php' => 'js/tracker.php',
+                    '<custom>' =>  Piwik::translate('TagManager_MatomoConfigurationMatomoJsEndpointCustom'),
```

Comment:
@EPNW-Eric I updated all the `<custom>` with `custom` and reverted the changes added in tests files and ran the tests again and that seems to work

![Screenshot from 2023-08-27 14-01-13](https://github.com/matomo-org/tag-manager/assets/14902277/72de9df9-51b7-4aed-b326-2255e60c19e3)